src/client/entity-editor/name-section/name-section.js

```diff
@@ -149,12 +157,24 @@ class NameSection extends React.Component {
 
 		const warnIfExists = !_.isEmpty(exactMatches);
 
+		if (!this.state.showButton && searchResults && searchResults.length > 5) {
+			this.setState({showButton: true});
+		}
+
+		if (this.state.showButton && searchResults && searchResults.length <= 5) {
+			this.setState({showButton: false});
+		}
+
+		if (this.state.showButton && searchResults === null) {
+			this.setState({showButton: false});
+		}
+
 
 		return (
-			<div>
+			<div className="margin-right-2">
 				<h2>{`What is the ${_.startCase(entityType)} called?`}</h2>
 				<Row>
-					<Col lg={{offset: 3, span: 6}}>
+					<Col>
```

Comment:
We still want these two columns to be collapsible on smaller screens
```suggestion
					<Col lg={{span:6}}>
```
Or maybe
```suggestion
					<Col md={{span:6}}>
```

Otherwise you get this on smaller screens:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/6179856/159757516-dd9b15c9-c8e1-4a58-a2fa-641416c8f39e.png">

Reply:
I agree